src/Processor/AuthorisedProcessor.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types=1);
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Webhook Module for PHP
+ *
+ * Copyright (c) 2021 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+
+namespace Adyen\Webhook\Processor;
+
+use Adyen\Webhook\EventCodes;
+use Adyen\Webhook\PaymentStates;
+
+class AuthorisedProcessor extends Processor implements ProcessorInterface
+{
+    public function process(): ?string
+    {
+        $state = $this->initialState;
+        $logContext = [
+            'eventCode' => EventCodes::AUTHORISED,
+            'originalState' => $state
+        ];
+        //Do nothing wait for the notification
```

Comment:
Which notification should we wait for in this case? Shouldn't this return a new payment state like `paid`?